packages/ember-simple-auth/package.json

```diff
@@ -62,7 +62,7 @@
     "ember-sinon": "~5.0.0",
     "ember-source": "~3.18.0",
     "ember-source-channel-url": "^2.0.1",
-    "ember-test-selectors": "^4.0.0",
+    "ember-test-selectors": "^6.0.0",
```

Comment:
this seems unrelated?

Reply:
I was working on getting the tests passing.  Looks like they are failing on the main branch so hopefully I can resolve here 👉 

https://github.com/simplabs/ember-simple-auth/pull/2319